pkg/constants/constants.go

```diff
@@ -125,6 +125,7 @@ var (
 	PredictorProtocolAnnotationKey                   = InferenceServiceInternalAnnotationsPrefix + "/predictor-protocol"
 	LocalModelLabel                                  = InferenceServiceInternalAnnotationsPrefix + "/localmodel"
 	LocalModelSourceUriAnnotationKey                 = InferenceServiceInternalAnnotationsPrefix + "/localmodel-sourceuri"
+	WorkerNodeSize                                   = KServeAPIGroupName + "/worker-node-size"
```

Comment:
After talking, and understanding that this is something internal, I think it should be:

```suggestion
	WorkerNodeSize                                   = InferenceServiceInternalAnnotationsPrefix + "/worker-node-size"
```


and make sure it is filtered out, similarly as in https://github.com/kserve/kserve/blob/158d90176804722fac0897386dae9b3f249468c8/pkg/controller/v1beta1/inferenceservice/components/predictor.go#L226-L229.

Reply:
oops forgot to remove this. I don't use it anymore so I will remove it.